lib/cocoapods/bazel/target.rb

```diff
@@ -47,14 +47,10 @@ def initialize(installer, pod_target, non_library_spec = nil, default_xcconfigs
       end
 
       def bazel_label(relative_to: nil)
-        package_basename = File.basename(package)
-        if package == relative_to
-          ":#{label}"
-        elsif package_basename == label
-          "@#{package_basename}"
-        else
-          "@#{package_basename}//:#{label}"
-        end
```

Comment:
Option: To make the output slightly prettier in most cases you could keep this logic as
```ruby
if pod_target.pod_name == label
  "@#{pod_target.pod_name}"
else
  "@#{pod_target.pod_name}//#{label}"
end
```
but it's not strictly necessary